script.js

```diff
@@ -10,3 +10,186 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
+  const projectImageMobile = document.createElement('img');
+  const countImage = document.createElement('img');
+  const countImage2 = document.createElement('img');
+  const canopyText = document.createElement('h3');
+  const backText = document.createElement('h3');
+  const yearText = document.createElement('h3');
+  const projectHeader = document.createElement('h2');
+  const projectBody = document.createElement('div');
+  const projectDescription = document.createElement('div');
+  const pDescription = document.createElement('p');
+  const languageUl = document.createElement('ul');
+  const projectBtn = document.createElement('button');
+
+  projectImageWeb.src = element.featuredImg;
+  projectImageWeb.classList.add('project-image-web');
+  projectImageWeb.alt = 'project-image';
+  projectImageMobile.src = element.featuredImgMobile;
+  projectImageMobile.classList.add('first-project-image');
+  projectImageMobile.alt = 'First-project';
+  projectHeader.classList.add('project-heading');
+  projectHeader.innerText = element.name;
+  canopyText.innerText = 'canopy';
+  canopyText.classList.add('content-text', 'canopy');
+  countImage.src = './Assets/vectors/Counter.svg';
+  countImage.classList.add('count-vector');
+  backText.innerText = 'Back end dev';
+  backText.classList.add('content-text');
+  countImage2.src = './Assets/vectors/Counter.svg';
+  countImage2.classList.add('count-vector');
+  yearText.innerText = '2018';
+  yearText.classList.add('content-text');
+  pDescription.innerText = 'A daily selection of privately personalized reads; no accounts or sign-ups required';
+  languageUl.classList.add('languages');
+  projectDiv.classList.add('project');
+  projectDiv.append(projectImageDiv, projectContentDiv);
+  projectImageDiv.classList.add('project-image');
+  projectImageDiv.append(projectImageMobile, projectImageWeb);
+  projectContentDiv.classList.add('project-content');
+  projectContentDiv.append(projectHeader, projectBody);
+  projectBody.classList.add('project-body');
+  projectBody.append(projectDescription, pDescription, languageUl, projectBtn);
+  projectDescription.classList.add('description');
+  projectDescription.append(
+    canopyText,
+    countImage,
+    backText,
+    countImage2,
+    yearText,
+  );
+
+  projects[i].technologies.forEach((tech, n) => {
+    const languageLi = document.createElement('li');
+    languageLi.classList.add('language-used');
+    languageUl.append(languageLi);
+    languageLi.innerText = projects[i].technologies[n];
+  });
+
+  projectContainer.appendChild(projectDiv);
+
+  projectBtn.classList.add('btn', 'project-link');
+  projectBtn.innerText = 'see project';
+  if (i % 2 !== 0) {
+    projectDiv.classList.add('card-reverse');
+  }
+});
+
+const modalContainer = document.querySelector('.modal-container');
+const modal = document.createElement('div');
+modalContainer.append(modal);
+modal.classList.add('modal');
+
+modal.innerHTML = `
```

Comment:
- [ ] [Optional] To improve users experience, please make sure that when scrolling down the popup window, the close button don't disappear from screen:
![image](https://user-images.githubusercontent.com/52635378/175614389-5907fcae-f656-41e0-b887-948855e78d01.png)